dev-docs/bidders/precisonat.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: bidder
+title: Precisonat
+description: Prebid Precisonat Bid Adapter
+biddercode: precisonat
+gvl_id: 874
+media_types: native
+tcfeu_supported: true
+usp_supported: true
+pbjs: true
+pbs: false
+prebid_member: true
+floors_supported: true
+safeframes_ok: true
+schain_supported: true
+userIds: sharedId
+deals_supported: false
+coppa_supported: true
+multiformat_supported: will-not-bid
+ortb_blocking_supported: true
+sidebarType: 1
+---
+
+### Modules
+
+SharedID: We need you to include the [SharedID module](/dev-docs/modules/userid-submodules/sharedid.html) in order to bid effectively on your inventory.
+
+### Registration
+
+The precisonat Bidding adapter requires setup before beginning. Please contact us at [tech@preciso.net]
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description         | Example       | Type     |
+|---------------|----------|---------------------|---------------|----------|
+| `publisherId` | required | Numeric Publisher ID <br>(as provided by Preciso)  | `'123AB123'`    | `string` |
+| `region`      | optional,recommended | 3 letter country code     | `'USA'` | `string` |
+| `bidFloor`    | optional,recommended | Minimum bid for this impression expressed in CPM (USD)  | `0.01`        | `float`  |
+| `pageType`    | optional,recommended | Kind of content present in the page   | `'homepage'`          | `String`     |
+| `currency`    | optional,recommended | type of currencies   |  `['USD']`   |  `string array`  |
+| `bcat`        | optional | List of blocked advertiser categories (IAB)   | `['IAB1-1']`          | `string array`    |
+| `badv`        | optional | Blocked Advertiser Domains| `'example.com'`   | `string array`| 
+
+Notes:
+
+- It is best to provide the global `bcat` and `badv` within the OpenRTB fields (below). When both methods are provided, ORTB values will be prioritized.
```

Comment:
Would also like for you to note that bidFloor would best be supplied by the Prebid Floors module and currency by the Currency module.